frontend/src/components/account/account.tsx

```diff
@@ -0,0 +1,16 @@
+import { AccountType } from "../../lib/types";
+import { getFormattedCurrency } from "../../lib/util";
+
+import styles from "./acc.module.css";
+
+export default function Account(props: AccountType) {
+  return (
+    <div className={styles.container}>
+      <div>Login({props.login})</div>
+      <div>Balance({getFormattedCurrency(props.balance)})</div>
+      <div>Equity({getFormattedCurrency(props.equity)})</div>
+      <div>Profit({getFormattedCurrency(props.profit)})</div>
+      <div>Server({props.server})</div>
```

Comment:
What is server here?

Reply:
![server](https://github.com/BensGitHub2022/MT5-Expert-Advisor/assets/97774893/fbc7c970-b11b-4d0d-bd5d-74d75c2088f7)